File: iOS/Pod/Classes/DoricContextHolder.m
//
//  DoricComponent.m
//  Doric
//
//  Created by pengfei.zhou on 2019/7/29.
//

#import "DoricContextHolder.h"

@implementation DoricContextHolder

- (instancetype)initWithContext:(DoricContext *)doricContext {
    if(self = [super init]){
        _doricContext = doricContext;
    }
    return self;
}

@end

File: Android/doric/src/main/java/pub/doric/utils/DoricContextHolder.java
package pub.doric.utils;

import pub.doric.DoricContext;

/**
 * @Description: com.github.penfeizhou.doric.utils
 * @Author: pengfei.zhou
 * @CreateDate: 2019-07-20
 */
public abstract class DoricContextHolder {
    private final DoricContext doricContext;

    public DoricContextHolder(DoricContext doricContext) {
        this.doricContext = doricContext;
    }

    public DoricContext getDoricContext() {
        return doricContext;
    }
}
